src/templates/kit/KnownAppBadge.js

```diff
@@ -1,19 +1,29 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import { AppBadge } from '@aragon/ui'
+import iconSvgAddressBook from './icons/address-book.svg'
 import iconSvgAgent from './icons/agent.svg'
+import iconSvgAllocations from './icons/allocations.svg'
+import iconSvgDotVoting from './icons/dot-voting.svg'
 import iconSvgFinance from './icons/finance.svg'
 import iconSvgFundraising from './icons/fundraising.svg'
 import iconSvgPayroll from './icons/payroll.svg'
+import iconSvgProjects from './icons/projects.svg'
+import iconSvgRewards from './icons/rewards.svg'
 import iconSvgTokens from './icons/token-manager.svg'
 import iconSvgVault from './icons/vault.svg'
 import iconSvgVoting from './icons/voting.svg'
 
 const KNOWN_ICONS = new Map([
+  ['address-book.aragonpm.eth', iconSvgAddressBook],
```

Comment:
Double checking (cc @Quazia), is this how Open Enterprise will be published?

Did we decide on using the main `aragonpm.eth` or `hatch.aragonpm.eth` or `open.aragonpm.eth`?

Reply:
We already published yesterday on rinkeby by using the main `aragonpm.eth` so this should be ready to go